sdk/eventhub/event-hubs/package.json

```diff
@@ -51,7 +51,7 @@
     "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
     "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
     "build:types": "downlevel-dts types/latest types/3.1",
-    "build": "tsc -p . && rollup -c 2>&1 && npm run extract-api && npm run build:types",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
```

Comment:
Why do some packages have "build:ts" and some don't?

Reply:
It's not a necessity to have the build:ts actually

Reply:
I agree it's not a necessity. 
If the whole purpose of this PR is to be consistent across packages, let's be consistent.

Reply:
Cool, removed that script.